src/Avalanche/Signer.cpp

```diff
@@ -108,8 +107,37 @@ BaseTransaction buildBaseTx(const Proto::SigningInput& input) noexcept {
     return structToBaseTx(txStruct);
 }
 
-Bitcoin::Proto::TransactionPlan Signer::plan(const Proto::SigningInput& input) noexcept {
-    return Bitcoin::Proto::TransactionPlan();
+Proto::TransactionPlan Signer::plan(const Proto::SigningInput& input) noexcept {
+    auto plan = Proto::TransactionPlan();
+
+    if (!input.has_simple_transfer_tx()) {
+        // not simple transaction
+        return plan;
+    }
+    auto tx = input.simple_transfer_tx();
+
+    plan.set_amount(tx.amount());
+    const auto n = tx.inputs_size();
+
+    // TODO proper selection based on amount -- is it really needed?
```

Comment:
```suggestion
    // TODO proper selection based on amount -- is it really needed?
```
I'm thinking reducing future UTXO chain integration effort, maybe we can add simple generic algo / plan for this?